### ISSUE TITLE
EPG Error

### ISSUE THREAD

**Issue by eric859** [Cigaras/IPTV.bundle#134]
Got an error with standard install, so I replaced the .pl with the latest updated xmltv_parser.pl that fixed the unicode issue.  Using both http link to xml as well as downloaded xml from same link I still get the following error:
```
2017-06-24 15:17:46,298 (b14) :  CRITICAL (core:574) - Exception in thread named 'GuideReloader' (most recent call last):
  File "C:\Program Files (x86)\Plex\Plex Media Server\Resources\Plug-ins-995f1dead\Framework.bundle\Contents\Resources\Versions\2\Python\Framework\components\runtime.py", line 1126, in _start_thread
    f(*args, **kwargs)
  File "C:\Users\htpc2\AppData\Local\Plex Media Server\Plug-ins\IPTV.bundle\Contents\Code\xmltv_parser.py", line 123, in GuideReloader
    LoadGuide()
  File "C:\Users\htpc2\AppData\Local\Plex Media Server\Plug-ins\IPTV.bundle\Contents\Code\xmltv_parser.py", line 83, in LoadGuide
    desc = programme.find('desc').text.decode("utf-8")
AttributeError: 'NoneType' object has no attribute 'decode'
```

**Comment by eric859** [comment]
Is a 2.2MB file too big for Plex?

**Comment by Acceptu** [comment]
I don't think so

**Comment by eric859** [comment]
Then I'm at a loss.  The xml file header says UTF-8.  It works with all the other software I've been testing with.  Just not this one.

**Comment by Acceptu** [comment]
I'll check my epg size and let you know what it is , if it's bigger maybe you can try mine to confirm

**Comment by eric859** [comment]
Thanks! Let me know.  I've attached mine in case you would be able to test it.  If not, no problem.
[xmltv.xml.txt](https://github.com/Cigaras/IPTV.bundle/files/1100039/xmltv.xml.txt)

**Comment by Cigaras** [comment]
@eric859 what error did You get before mixing internal plug-in files?

**Comment by yurra2** [comment]
Hi All!
Still have the issue on 2.1.6
Using xmltv epg from here http://programtv.ru/xmltv.xml.gz

**Comment by Cigaras** [comment]
@yurra2, Your issue is fixed by @tylerrichey (0bc3e16e2450017a45ee3f46630d03bbf3a26cd7), latest version 2.1.6 is available in [releases](https://github.com/Cigaras/IPTV.bundle/releases) section.

However I see Your program guide is quite big, when I try to manually load it in Plex Web client it says that the server is not responding because it takes more time for server to load the xml than client is willing to wait. But server loads the whole xml without issues disregarding the error the client shows. For now I do not have a solution for that client side error message... Maybe I will move manual loading to the background where automatic loading is running now, someday in the future...
